### PR TITLE
Modulesync 0.12.5 & Release 0.1.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## 2016-08-19 Release 0.1.0
```
Comment (igalic) [inline CHANGELOG.md]: why not commit and release 1.0.0?

Comment (bastelfreak) [inline CHANGELOG.md]: I don't like to release a 1.0.0 without any spec tests

Comment (alexjfisher) [inline CHANGELOG.md]: README could probably do with some sort of introductory paragraph before a 1.0.0?

Comment (igalic) [inline CHANGELOG.md]: i'd rather make a good release than just a release :P

**Pull request merged**